modules/gateway/controllers/publicdomain/populate/destinationrule.go

```diff
@@ -17,7 +17,7 @@ func DestionationRule(pd *adminv1.PublicDomain, systemNamespace string, os *obje
 			Host: fmt.Sprintf("%s.%s.svc.cluster.local", name, systemNamespace),
 			Subsets: []v1alpha3.Subset{
 				{
-					Name: "latest",
```

Comment:
Does this name change do anything?

Reply:
It doesn't, but would like a special name we can tell so that we don't program in the following place 
```
if dests[0].Subset != "" && dests[0].Subset != "cm-acme" {
```